packages/opentelemetry-core/src/trace/GlobalTracer.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as types from '@opentelemetry/types';
+import { NoopTracer } from './NoopTracer';
+
+const noopTracer = new NoopTracer();
+let _globalTracer: types.Tracer | null = null;
+
+export class GlobalTracerDelegate implements types.Tracer {
+  getCurrentSpan(): types.Span {
+    const tracer = _globalTracer || noopTracer;
+    // tslint:disable-next-line:no-any
+    return tracer.getCurrentSpan.apply(tracer, arguments as any);
+  }
+
+  startSpan(name: string, options?: types.SpanOptions | undefined): types.Span {
+    const tracer = _globalTracer || noopTracer;
+    // tslint:disable-next-line:no-any
+    return tracer.startSpan.apply(tracer, arguments as any);
+  }
+
+  withSpan<T extends (...args: unknown[]) => unknown>(
+    span: types.Span,
+    fn: T
+  ): ReturnType<T> {
+    const tracer = _globalTracer || noopTracer;
+    // tslint:disable-next-line:no-any
+    return tracer.withSpan.apply(tracer, arguments as any) as ReturnType<T>;
+  }
+
+  recordSpanData(span: types.Span): void {
+    const tracer = _globalTracer || noopTracer;
+    // tslint:disable-next-line:no-any
+    return tracer.recordSpanData.apply(tracer, arguments as any);
+  }
+
+  getBinaryFormat(): unknown {
```

Comment:
Is it possible change `unknown` to more specific type i.e. `BinaryFormat`? Same for below method.

Reply:
Related to this todo. Looks like this was implemented in #74, so it should also be updated in the interface as well
https://github.com/open-telemetry/opentelemetry-js/blob/39ca4699eec563aef958ed04a585574d9140f7f3/packages/opentelemetry-types/src/trace/tracer.ts#L77-L79

Reply:
Hmm right, feel free to update here. I will open another PR to update the `Tracer` interface.

Reply:
This has been done in https://github.com/open-telemetry/opentelemetry-js/pull/125, please rebase.